source/common/local_info/local_info_impl.h

```diff
@@ -10,8 +10,9 @@ namespace LocalInfo {
 class LocalInfoImpl : public LocalInfo {
 public:
   LocalInfoImpl(const envoy::api::v2::core::Node& node,
-                Network::Address::InstanceConstSharedPtr address, const std::string zone_name,
-                const std::string cluster_name, const std::string node_name)
+                const Network::Address::InstanceConstSharedPtr& address,
+                const std::string zone_name, const std::string cluster_name,
```

Comment:
While you are in here, can you either pass these by const ref or make them non-const and move them?